src/controllers/accommodation.js

```diff
@@ -0,0 +1,52 @@
+import models from '../db/models';
+import dotenv from 'dotenv';
+dotenv.config();
+
+export default class accomodationFacility {
+  static async getAllAccommodations(req,res){
+    try{
+    const accommodations = await models.Accommodation.findAll();
+        return res.status(200).json({status:200, data:accommodations});
+    }catch(error){
```

Comment:
Please fix your ESLint in order to have 2 spaces of indentation.

Reply:
I have managed to configure my ESLint